sklearn/multiclass.py

```diff
@@ -504,19 +520,6 @@ def _more_tags(self):
     def _first_estimator(self):
         return self.estimators_[0]
 
-    @property
-    def n_features_in_(self):
-        # For consistency with other estimators we raise a AttributeError so
-        # that hasattr() fails if the OVR estimator isn't fitted.
-        try:
-            check_is_fitted(self)
-        except NotFittedError as nfe:
-            raise AttributeError(
-                "{} object has no n_features_in_ attribute."
-                .format(self.__class__.__name__)
-            ) from nfe
-        return self.estimators_[0].n_features_in_
```

Comment:
@NicolasHug, reading the slep of n_features_in_, it says that it can be defined either as a property or in fit. Is there a reason you defined a property here ? I ask because it looks more complicated this way, compared to the 2 lines in fit. If it's just a matter of taste, I can revert :)

Reply:
I can't remember TBH, I probably just copy/pasted from other meta-estimators. It sounds fine to me to set it in fit instead, as long as CI is green :)